styling.css

```diff
@@ -837,6 +854,194 @@
+    left: 50%;
+    transform: translate(-47%, -50%);
+    display: none;
+  }
+
+  .heading-section {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 20px;
+    padding-bottom: 30px;
+  }
+
+  .modalH2 {
+    font-family: 'Crete Round', sans-serif;
+    font-size: 1.5rem;
+    color: #091e42;
+    margin-left: 30px;
+  }
+
+  .Xbutton {
+    font-size: 30px;
+    font-weight: 900;
+    margin-right: 20px;
+    cursor: pointer;
+  }
+
+  .modalListHold {
+    display: flex;
+    gap: 5px;
+    padding-left: 0px;
+    margin-left: 3.5vw;
+    width: 220px;
+    height: fit-content;
+    overflow-x: hidden;
+  }
+
+  .modalListHoldTop {
+    display: flex;
+    gap: 5px;
+    padding-left: 0px;
+    margin-left: 3.5vw;
+    width: fit-content;
+    height: fit-content;
+    overflow-x: hidden;
+  }
+
+  .modalListItem {
+    border-radius: 0px;
+    border: 2px solid #aaa;
+    font-family: 'Inter', sans-serif;
+    font-size: 1.063rem;
+    font-weight: 800;
+    text-align: left;
+    padding: 5px;
+  }
+
+  .rest-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: auto;
+  }
+
+  .modalImage {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    width: 360px;
+    height: 214px;
+  }
+
+  .modalImageOther {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    width: 360px;
+    height: 214px;
+  }
+
+  .restArticle {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    padding-top: 0px;
+  }
+
+  .modalDescription {
+    font-family: 'Inter', sans-serif;
+    font-weight: 900;
+    font-size: 1.188rem;
+    color: #091e42;
+    column-gap: 10px;
+    width: 360px;
+    height: fit-content;
+    padding-bottom: 10px;
+  }
+
+  .modalbuttonsholder {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    column-gap: 60px;
+    margin-top: 10px;
+  }
+
+  .modalbutton1 {
+    background-color: #c46710;
+    width: 150px;
+    border: 0;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    margin-left: 20px;
+  }
+
+  .modalbutton1:hover {
+    background-color: #ceb49d;
+  }
+
+  .modalbutton1:active {
+    background-color: #973a04;
+  }
+
+  .modalbutton1 > a {
+    color: white;
+    font-family: 'Inter', sans-serif;
+    display: flex;
+    column-gap: 10px;
+    justify-content: center;
+  }
+
+  .modalbutton2 {
+    background-color: #c46710;
+    width: 165px;
+    border: 0;
+    padding-top: 15px;
+    padding-bottom: 15px;
+    margin-right: 33px;
+  }
+
+  .modalbutton2:hover {
+    background-color: #ceb49d;
+  }
+
+  .modalbutton2:active {
+    background-color: #973a04;
+  }
+
+  .modalbutton2 > a {
+    width: fit-content;
+    color: white;
+    font-family: 'Inter', sans-serif;
+    display: flex;
+    column-gap: 10px;
+    justify-content: center;
+  }
```

Comment:
- [ ] Great job so far. However, there're few things to work on before you move to the next milestone.
   -  The window popup  on the mobile version does not appear as we have it on the figma design. 
   - The `X` icon to close the modal is not showing on the popup window.
   - The arrangement of the list of tools used to build the project are not properly layout as we have them on the figma design.
   - There's a small horizontal scroll bar at the bottom of the page.
 kindly attend to all these observations and fix them.


### You Design
![your-mobile-windo-popup](https://user-images.githubusercontent.com/26597432/233429002-9a71dc12-18fd-4237-ba6e-3f4cb38ee985.png)

### Screen Recording
[app-behave.webm](https://user-images.githubusercontent.com/26597432/233429136-05b6381b-2b61-49ec-9426-620b77482bdd.webm)

### Figma Design
![temp-mobile-window-popup](https://user-images.githubusercontent.com/26597432/233429186-4fc2a861-6568-4d6c-a6b5-3015184a6577.png)

Reply:
### **As per my previous CR 🔍  i was requested ♻️  to make some changes like :**
**-make better design for mobile view to fit figma design🎨**
**-Remove the horizontal scroll bar 📜 to improve ux.**

as per my last commit 🟢  to the pull request i have fixed  🧑‍🔧  all the issues above. 🏅  
I hope to hear from you my CR 🤺  soon!.